src/pages/changePassword.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { ReactElement } from 'react';
+import { NextPageContext } from 'next';
+import { parseCookies } from 'nookies';
+import { TwoThirdsLayout } from '../layout/Layout';
+import ErrorSummary from '../components/ErrorSummary';
+import FormElementWrapper from '../components/FormElementWrapper';
+import { USER_COOKIE } from '../constants';
+import { CustomAppProps, ErrorInfo } from '../interfaces';
+import CsrfForm from '../components/CsrfForm';
+
+const title = 'Change Password - Fares data build tool';
+const description = 'Change Password page of the Fares data build tool';
+
+interface ChangePasswordProps {
+    errors: ErrorInfo[];
+}
+
+const ChangePassword = ({ errors, csrfToken }: ChangePasswordProps & CustomAppProps): ReactElement => {
+    return (
+        <TwoThirdsLayout title={title} description={description} errors={errors}>
+            <div className="govuk-grid-row">
+                <div className="govuk-grid-column-two-thirds">
+                    <CsrfForm action="/api/changePassword" method="post" csrfToken={csrfToken}>
+                        <>
+                            <ErrorSummary errors={errors} />
+                            <div className={`govuk-form-group ${errors.length > 0 ? 'govuk-form-group--error' : ''}`}>
+                                <div className="govuk-fieldset" aria-describedby="change-password-page-heading">
+                                    <legend className="govuk-fieldset__legend govuk-fieldset__legend--l">
+                                        <h1 className="govuk-fieldset__heading" id="change-password-page-heading">
+                                            Change your password
+                                        </h1>
+                                    </legend>
+                                    <p className="govuk-hint hint-text" id="change-password-page-hint">
+                                        Your password should be at least 8 characters long.
+                                    </p>
+
+                                    <div className="govuk-form-group">
+                                        <label className="govuk-label" htmlFor="old-password">
+                                            Old password
```

Comment:
Would previous password make more sense than old? I can't recall seeing 'old' used in this type of form.

Reply:
Hmm possibly. Maybe one for a user research session? I thought old made sense with the other inputs referring to a 'new' password. But I agree it doesn't make the most sense to have this.